docs/manual/external-tools.tex

```diff
@@ -537,6 +538,34 @@ \section{Javac compiler\label{javac-installation}}
 % \end{itemize}
 
 
+\section{Lombok\label{lombok}}
+
+Project Lombok (\url{https://projectlombok.org/}) is ``a java library that
+automatically plugs into your editor and build tools, spicing up your java.''
```

Comment:
Rather than "spicing up your Java" (which doesn't convey any content), I would say that it generates methods (getters/setters/equals/etc.) so you don't have to write them explicitly.

Reply:
This is a quote from the Lombok website.

Reply:
True, but it's still not helpful and should be improved.